core/src/modules/ops/destination.ts

```diff
@@ -912,13 +913,41 @@ export namespace DestinationOps {
     success: boolean;
     error: Error;
   }> {
-    const _exports: Export[] = []; // only ones we are sending
+    const _exports: Export[] = [];
+
     for (const _export of givenExports) {
-      // TODO: maybe should recalcuate hasChanges and from current mostRecent
-      if (!_export.hasChanges) {
-        await _export.complete(); // do not include exports with hasChanges=false
+      const { isLocked, lockedBy } = await getLock(
+        `${_export.recordId}:${_export.destinationId}`,
+        _export.id,
+        null,
+        "export"
+      );
+
+      if (isLocked) {
+        const lockingExport = await Export.findById(lockedBy);
```

Comment:
Perhaps this should be 

```ts
const mostRecentExport = await Export.findOne({where: {recordId: export.recordId, destinationId: export.destinationId}, order: ['createdAt', 'ASC']})
```

Then, you can be sure you have the most recent export and you can compare if `mostRecentExport.id === _export.id`.  Either "I" am the most recent export or not

Reply:
I'm wondering if we want `startedAt` or `createdAt` here... `startedAt` has to do with the batch it's in I believe.  In testing, I'm seeing two exports may have the same `startedAt` while they have differing `createdAt`s.